hack/update-codegen.sh

```diff
@@ -119,6 +121,8 @@ ${informergen} \
   --listers-package k8s.io/client-go/listers \
   "$@"
 
```

Comment:
please check that verify-codecgen.sh works fine

Reply:
Yeah I don't have any verification yet. I mostly wanted to get what I had up for discussion sooner rather than later.

Reply:
looks good so far.